packages/ts/lit-form/src/BinderNode.ts

```diff
@@ -334,7 +438,11 @@ export class BinderNode<M extends AbstractModel = AbstractModel> extends EventTa
 
   prependItem(item?: Value<ArrayItemModel<M>>): void {
     if (this.#isArray()) {
-      const itemValueOrEmptyValue = item ?? this.model[_createEmptyItemValue]();
+      const itemValueOrEmptyValue =
+        item ??
+        (this.model instanceof BinderArrayModel
+          ? this.model[_createEmptyItemValue]()
+          : this.model[$itemModel][$defaultValue]);
```

Comment:
Hmm, what about creating accessor functions to avoid checking this all the time? E.g.:
```ts
item ?? createEmptyValue(this.model);

function createEmptyValue(model: M) {
  // Do all the necessary checks
}
```